src/migration/ember-addon/steps/augment-options.js

```diff
@@ -67,33 +75,21 @@ function analyzePackageManager(options) {
 }
 
 function deriveAddonLocation(addonPackage) {
-  if (!addonPackage.name) {
-    throw new SyntaxError(
-      'ERROR: In package.json, the package name is missing.'
-    );
+  // Package is not scoped
+  if (!addonPackage.name.includes('/')) {
+    return addonPackage.name;
   }
 
-  if (!addonPackage.version) {
```

Comment:
After merging #3, I realized that I shouldn't check the package name and version in `deriveAddonLocation()`. After all, the package version is unrelated to the addon location.